From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 1 Jun 2018 14:37:13 +0100
Subject: hwmon: (k10temp) Make function get_raw_temp static
Git-commit: fb8eefd3b4e6f79e0930fffff6640744699c6f1d
Patch-mainline: v4.18
References: FATE#327735

The function get_raw_temp is local to the source and does not need to
be in global scope, so make it static.

Cleans up sparse warning:
drivers/hwmon/k10temp.c:149:14: warning: symbol 'get_raw_temp' was not
declared. Should it be static?

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
Acked-by: Jean Delvare <jdelvare@suse.de>

diff --git a/drivers/hwmon/k10temp.c b/drivers/hwmon/k10temp.c
index d3fae5a8e508..17c6460ae351 100644
--- a/drivers/hwmon/k10temp.c
+++ b/drivers/hwmon/k10temp.c
@@ -146,7 +146,7 @@ static void read_tempreg_nb_f17(struct pci_dev *pdev, u32 *regval)
 		     F17H_M01H_REPORTED_TEMP_CTRL_OFFSET, regval);
 }
 
-unsigned int get_raw_temp(struct k10temp_data *data)
+static unsigned int get_raw_temp(struct k10temp_data *data)
 {
 	unsigned int temp;
 	u32 regval;
